pages/linux/protontricks.md

```diff
@@ -18,3 +18,11 @@
 - Run a command within a game's installation directory:
 
 `protontricks -c {{command}} {{appid}}`
+
+- List all installed games:
+
+`protontricks -l`
+
+- Search for a game's App ID by name:
```

Comment:
```suggestion
- [s]earch for a game's App ID by name:
```

Please use mnemonics as much as possible. If you don't know which mnemonic to use we can help. ;)

Reply:
> Please use mnemonics as much as possible. If you don't know which mnemonic to use we can help. ;)

Sure, I will try to use more mnemonics in future, I was just confused about whether to use them in commands with single arguments.